### ISSUE TITLE
Can't validate

### ISSUE THREAD

**Issue by DavidTortolo** [xnimorz/masked-input#66]
Im trying to validate a field masked like a creditcard (mask="0000-0000-0000-0000") and I check if the input value is null. Which isn't, because is ____-____-____-____. So it should be a way to check if the value is full filling the mask.

**Comment by xnimorz** [comment]
Hey @DavidTortolo . 

Unfortunately, I don't work on the package actively, I can suggest to subscribe to `onValueChanged` event, and use either `value` field from that event / track changes in it

**Comment by DavidTortolo** [comment]
what I did is make a custom validation method for each field I want. For example if I use t he mask with underscore for credit card, to check if it was fullfilled, I checked that no underscore were present. 
As you're not working on this actively do you suggest to move to another solution?

**Comment by xnimorz** [comment]
> do you suggest to move to another solution?

Yes, I don't expect to have time to support this particular project in foreseen future. 
So, the right answer is either this project will get another maintainer, or it's better to move to another library. However I don't have a good candidate.
